jetstream/engine/token_utils.py

```diff
@@ -349,6 +349,11 @@ def bos_id(self) -> int:
     """ID of the BOS token."""
     return self.vocab.bos_id
 
+  @property
+  def stop_tokens(self) -> set[int]:
+    """ID of the stop token."""
+    return {self.eos_id, self.pad_id}
+
 
```

Comment:
@JoeZijunZhou  What is TikToken? (some tokenizer from tiktok)

Reply:
it's llama3's tokenizer (apperently made by OpenAI)